packages/azure_functions/manifest.yml

```
@@ -0,0 +1,86 @@
+format_version: 2.5.1
+name: azure_functions
+title: "Azure Functions"
+version: 0.1.0
+source:
+  license: "Elastic-2.0"
+description: "Get metrics and logs from Azure Functions"
+type: integration
+categories:
+  - azure
+  - cloud
+conditions:
+  kibana.version: "^8.6.1"
```

```suggestion
  kibana.version: "^8.8.1"
```

We need at least 8.8.1 to use the JSON santitization feature added in this version.

---

Good point, will change that.